src/docfx/lib/schema/JsonSchemaValidator.cs

```diff
@@ -366,16 +376,53 @@ private void ValidateEnumDependencies(JsonSchema schema, JObject map, List<Error
                             if (allowLists.TryGetValue(dependentFieldValue, out var allowList) &&
                                 Array.IndexOf(allowList, fieldValue) == -1)
                             {
-                                errors.Add(Errors.InvalidPairedAttribute(JsonUtility.GetSourceInfo(map), fieldName, fieldValue, dependentFieldName, dependentFieldValue));
+                                errors.Add(GenerateAdditionalError(schema, dependentFieldName, Errors.InvalidPairedAttribute(JsonUtility.GetSourceInfo(map), fieldName, fieldValue, dependentFieldName, dependentFieldValue), map));
```

Comment:
you can transform the errors at (line 29) after all errors are collected, can add a `PropertyName` property to `Error`, with that, you don't need to modify `additionalErrors`

Reply:
But addtionalErrors is not always at root level, so property name maybe not enough to for looking up

Reply:
okok, this is more simple